### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by fed1k**
# Hi Code Reviewer🙋‍♂️. I followed the requirements below:
  * Initialise React app.
  * The building blocks of your app should be set as re-usable components. Create a directory for your components.
  * Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
 
  ## Books - the default view:
  * Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
 * Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
  * Should have a form for adding a book (no functionality yet).
  * Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
## Categories:
  * Should display a button "Check status" only.
  * Styling is not required at this point.  ( Add styling for only navbar )
  
## Happy Reviewing🐱‍👤

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @fed1k, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] README file should be descriptive
- all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo. Kindly add all the neccesary details.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  {
+    title: 'Capital in the twentry-first Century',
+    author: 'Suzanne Collins',
+  },
+];
+const Books = () => (
+  <div>
+    <Book title={arrayOfBooksInfo[0].title} author={arrayOfBooksInfo[0].author} />
+    <Book title={arrayOfBooksInfo[1].title} author={arrayOfBooksInfo[1].author} />
+    <Book title={arrayOfBooksInfo[2].title} author={arrayOfBooksInfo[2].author} />
```
Comment (mmsesay) [inline src/components/Books.js]: - [OPTIONAL] To simplify this approach, you could iterate through the `arrayOfBooksInfo`

Something like:
```js
arrayOfBooksInfo.map((book) => (
    <Book title={book.title} author={book.author} />
));
```

**Review by mmsesay — APPROVED** [review]
Hi @fed1k, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (fed1k) [reply src/components/Books.js]: @mmsesay Hi bro. Thank you for suggestion. I will implement that then merge. Thanks again👏

**Pull request merged**